runtime/doc/vim_diff.txt

```diff
@@ -28,6 +28,10 @@ these differences.
 ==============================================================================
 2. Option defaults					  *nvim-option-defaults*
 
+- syntax highlighting is enabled by default (can be disabled with the "-u NONE"
+  cmdline argument)
+- filetype-related plugins and scripts are enabled by default (can be disabled
+  with the "-u NONE" cmdline argument)
```

Comment:
I put "plugins and scripts" because this loads the filetype and ftplugin files plus the indent file, and in my mind only the second(ftplugin) defines proper "plugins".